Fixes failure to build tests with USE="-cxx" (default).
http://developer.intra2net.com/git/?p=libftdi;a=patch;h=11a50ae5b80b3e03694a19e84513345d0794e563;hp=6dd18122a81e6030dd239391df6ffde8cb8c59c7
From 11a50ae5b80b3e03694a19e84513345d0794e563 Mon Sep 17 00:00:00 2001
From: Thomas Jarosch <thomas.jarosch@intra2net.com>
Date: Mon, 24 Aug 2020 19:27:22 +0200
Subject: [PATCH 1/1] Fix building unit tests without FTDIPP

Needed to run the baudrate unit tests.

Probably another fallout from:

****************************
commit 0209a3633dc877a577af07d883eb5059e22f6a91

cmake: do not check for g++ when FTDIPP is disabled
****************************
---
 CMakeLists.txt |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3b0b87c..58f664a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -150,6 +150,7 @@ if ( EXAMPLES )
 endif ()
 add_subdirectory(packages)
 if ( BUILD_TESTS )
+  project(libftdi1 C CXX)
   add_subdirectory(test)
 endif ()
 
-- 
1.7.1

